packages/babel-types/src/validators/isReferenced.js

```diff
@@ -7,60 +7,50 @@ export default function isReferenced(node: Object, parent: Object): boolean {
     // yes: object::NODE
     // yes: NODE::callee
```

Comment:
Mostly nit but I noticed that sometimes we are grouping `return false` and some are not. Maybe could you regroup them?

Reply:
Done.